From b8308a495f853d066c5c0e5d2257a070b033f626 Mon Sep 17 00:00:00 2001
From: Kam Nasim <kam.nasim@windriver.com>
Date: Tue, 5 Jul 2016 16:46:28 -0400
Subject: [PATCH] CGTS-4280: Fixing mismatched permission on dnsmasq.conf which
 was set to 0640 when created from config_controller (controller-0) but was at
 0644 on controller-1 through application of this manifest.

---
 packstack/puppet/modules/dnsmasq/manifests/params.pp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packstack/puppet/modules/dnsmasq/manifests/params.pp b/packstack/puppet/modules/dnsmasq/manifests/params.pp
index 6dd5b96..6129b57 100644
--- a/packstack/puppet/modules/dnsmasq/manifests/params.pp
+++ b/packstack/puppet/modules/dnsmasq/manifests/params.pp
@@ -51,7 +51,7 @@ class dnsmasq::params {
   }
 
   $config_file_mode = $::operatingsystem ? {
-    default => '0644',
+    default => '0640',
   }
 
   $config_file_owner = $::operatingsystem ? {
-- 
1.8.3.1

